polaris-react/src/components/ColorPicker/ColorPicker.stories.tsx

```diff
@@ -37,3 +37,27 @@ export function WithTransparentValueFullWidth() {
 
   return <ColorPicker fullWidth onChange={setColor} color={color} allowAlpha />;
 }
+
+export function WithHexColorTextField() {
+  const [color, setColor] = useState({
+    hue: 300,
+    brightness: 1,
+    saturation: 0.7,
+    alpha: 0.7,
+  });
+
+  return <ColorPicker onChange={setColor} color={color} showTextField />;
+}
+
+export function WithHexColorTextFieldFullWidth() {
+  const [color, setColor] = useState({
+    hue: 300,
+    brightness: 1,
+    saturation: 0.7,
+    alpha: 0.7,
+  });
+
+  return (
+    <ColorPicker onChange={setColor} color={color} showTextField fullWidth />
```

Comment:
I'm not sure I love this API. `showTextField` seems too rigid and generic. What if teams want to be able paste and parse rgb, rgba, OKLab, OKLCH etc...

Reply:
My initial idea was to start with the basic functionality that would cover the needs of small projects (like my own - Shopify Forms) and expand it to support other use cases afterwards. There are way more complex use cases like [this](https://github.com/Shopify/online-store-web/blob/ad6920a17fd2ce82d48805e0b53311c6c3251cba/packages/%40editor/color-utilities/index.ts#L319), that supports hex, rgb and even color names (the list is maintained [here](https://github.com/Shopify/online-store-web/blob/ad6920a17fd2ce82d48805e0b53311c6c3251cba/app/components/RichTextEditor/components/ColorPicker/utilities/nameHexMap.ts)).

Although it would be too big for a first version, the support to other use cases can come in another PR.